package.json

```diff
@@ -82,6 +82,7 @@
     "sass-lint": "1.5.1",
     "sass-loader": "2.0.1",
     "scratchr2_translations": "git://github.com/LLK/scratchr2_translations.git#master",
+    "selenium-webdriver": "^2.44.0",
```

Comment:
Please remove the `^` to match the other dependencies. Info on what it means: https://docs.npmjs.com/getting-started/semantic-versioning. And you can set up your npm to never include it with `npm config set save-prefix=''`: https://docs.npmjs.com/misc/config#save-prefix